guide/src/guide/reading.md

```diff
@@ -56,9 +56,24 @@ Clicking a highlighted word or pressing the <kbd>Escape</kbd> key will remove th
 mdBook books are often used for programming projects, and thus support highlighting code blocks and samples.
 Code blocks may contain several different icons for interacting with them:
 
+<style>
+.light .table-wrapper .clip-button, .rust .table-wrapper .clip-button {
+    --copy-button-filter: initial;
+}
+.coal .table-wrapper .clip-button {
+    --copy-button-filter: brightness(0) saturate(100%) invert(72%) sepia(9%) saturate(401%) hue-rotate(167deg) brightness(90%) contrast(84%);
```

Comment:
Since it's an icon, we need to cheat a bit to change its color. To do so, we rely on the `filter` CSS rule. There are converters only to achieve this result. :)